includes/class-wc-payments-blocks-payment-method.php

```diff
@@ -52,13 +52,17 @@ public function is_active() {
 	 * @return string[] A list of script handles.
 	 */
 	public function get_payment_method_script_handles() {
-		WC_Payments_Utils::enqueue_style(
-			'wc-blocks-checkout-style',
-			plugins_url( 'dist/blocks-checkout.css', WCPAY_PLUGIN_FILE ),
-			[],
-			'1.0',
-			'all'
-		);
+
+		if ( ( is_cart() || is_checkout() || is_product() ) ) {
```

Comment:
Is there any reason not to just exit this entire function if we are not on one of these three pages? Presumably we don't need to be enqueuing Stripe and blocks checkout scripts on pages other than the aforementioned, no?

Reply:
From what I understand, the `IntegrationInterface` handles actually enqueueing the scripts at the appropriate time/place. We just can't use it the same way for stylesheets because it'll break the script enqueue.

Reply:
Cool. AFAICT, can't see these scripts on other pages, so presumably all good here. 👍